src/mocks/CatalogueItems.json

```diff
@@ -27,6 +27,7 @@
     "days_to_rework": null,
     "drawing_number": null,
     "drawing_link": null,
+    "expected_lifetime_days": null,
```

Comment:
Can you change the expected life time values in this file.  So most of catalogue item documents have an expected lifetime (around 90% of the documents ) and Can the range of lifetime between 0.25 year to 4 years.

Reply:
I would say 0.5 years to 10 years would be a decent range.

Reply:
Is there a script that generates the mock data on the front end? Can't seem to find one.

Reply:
No there isn't.

Reply:
I did at one point use the backend one to generate mock data here but doing that now would effect the existing tests. The simplest thing to do would be to write some sort of typescript/python function that reads the json, randomly inserts them and then outputs the new json again. ChatGPT may be useful for this.

Reply:
Ah, yeah that's good idea. I spent a bit of time trying to get a regex pattern for the find and replace function in VSCode to work, but I ended up just manually changing it.